.gitignore

```diff
@@ -1,5 +1,4 @@
 /node_modules
-dist
```

Comment:
You will need to add `dist` back into the file, so we don't have to move the `dist` folder around.

Reply:
Added back